Resources/Prototypes/_SpaceCats/Entities/Markers/Spawners/Random/posters.yml

```
@@ -0,0 +1,23 @@
+- type: entity
+  parent: MarkerBase
+  id: HalloweenPosterRandom
+  name: random helloween wall decor spawner
+  #decription: random helloween wall decor spawner
+  suffix: Halloween
+  components:
+  - type: Sprite
+    layers:
+      - state: red
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)